test/automigrate.test.js

```diff
@@ -42,10 +44,17 @@ describe('sql connector', function() {
   });
 
   it('automigrate reports errors for models not attached', function(done) {
-    ds.automigrate(['m1', 'm2'], function(err) {
+    ds.automigrate(['m1', 'm3'], function(err) {
```

Comment:
any reason change `m2` to `m3`?

Reply:
Previously there was only a single table `m1` attached and `m2` was un-attached.   The new test (line #55) expects two tables created in a series.   So I refactored two tables `m1` and `m2` to be attached by default, and here to look for an un-attached table `m3`.

Reply:
Thank you for the explanation, fair enough.